pom.xml

```diff
@@ -24,23 +24,57 @@
 
   <properties>
     <java.version>1.8</java.version>
+    <slf4j.version>1.6.6</slf4j.version>
   </properties>
 
   <dependencies>
     <dependency>
       <groupId>org.springframework.boot</groupId>
       <artifactId>spring-boot-starter-batch</artifactId>
+        <exclusions>
+          <exclusion>
+            <groupId>org.slf4j</groupId>
+            <artifactId>log4j-over-slf4j</artifactId>
+          </exclusion>
+          <exclusion>
+            <groupId>ch.qos.logback</groupId>
+            <artifactId>logback-classic</artifactId>
+          </exclusion>
+        </exclusions>
     </dependency>
     <dependency>
-        <groupId>org.springframework.boot</groupId>
-        <artifactId>spring-boot-starter-test</artifactId>
+      <groupId>org.springframework.boot</groupId>
+      <artifactId>spring-boot-starter-test</artifactId>
     </dependency>
     <dependency>
-        <groupId>org.springframework.batch</groupId>
-        <artifactId>spring-batch-test</artifactId>
+      <groupId>org.springframework.batch</groupId>
+      <artifactId>spring-batch-test</artifactId>
+    </dependency>
+    <!-- slf4j -->
+    <dependency>
+      <groupId>org.slf4j</groupId>
+      <artifactId>slf4j-log4j12</artifactId>
+      <version>${slf4j.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>org.slf4j</groupId>
+      <artifactId>slf4j-api</artifactId>
+      <version>${slf4j.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>com.github.angelicaochoa</groupId>
```

Comment:
shouldn't this be from a public repo? (maybe after we merge the cbioportal PR)
Also, the pipelines repo pulls in core in a slightly different way .. not a checksum version id, but a symbolic one. Should we use the same approach in both places? (maybe we will -- I have not looked at the related pipelines PR yet)

Reply:
@sheridancbio this should def point to public cbioportal repo once the PR is fixed

I think it would be good to have pipelines pull core repo in the same way. Unless there's a good reason for using symbolic links over hardcoded versions I'd say let's use jitpack there as well.